### PR TITLE
Replace device with session in UI text

### PR DESCRIPTION
**Pull request by jryans**
This replaces "device" with "session" throughout user-visible text at the React
layer. Variable names and comments are left as-is for now.

### CODE REVIEW

File: src/Analytics.js [hunk 1]
```diff
@@ -272,7 +272,7 @@ class Analytics {
                 ),
             },
             { expl: _td('Your User Agent'), value: navigator.userAgent },
-            { expl: _td('Your device resolution'), value: resolution },
```
Comment (t3chguy) [inline src/Analytics.js]: this should remain as device

File: src/components/structures/auth/ForgotPassword.js [hunk 2]
```diff
@@ -358,9 +358,9 @@ export default createReactClass({
         return <div>
             <p>{_t("Your password has been reset.")}</p>
             <p>{_t(
-                "You have been logged out of all devices and will no longer receive " +
+                "You have been logged out of all sessions and will no longer receive " +
                 "push notifications. To re-enable notifications, sign in again on each " +
-                "device.",
+                "session.",
```
Comment (t3chguy) [inline src/components/structures/auth/ForgotPassword.js]: this should also be device

File: src/components/structures/auth/SoftLogout.js [hunk 3]
```diff
@@ -212,8 +212,8 @@ export default class SoftLogout extends React.Component {
         let introText = null; // null is translated to something area specific in this function
         if (this.state.keyBackupNeeded) {
             introText = _t(
-                "Regain access to your account and recover encryption keys stored on this device. " +
-                "Without them, you won’t be able to read all of your secure messages on any device.");
+                "Regain access to your account and recover encryption keys stored on this session. " +
```
Comment (t3chguy) [inline src/components/structures/auth/SoftLogout.js]: device

File: src/components/structures/auth/SoftLogout.js [hunk 4]
```diff
@@ -212,8 +212,8 @@ export default class SoftLogout extends React.Component {
...
+                "Without them, you won’t be able to read all of your secure messages on any session.");
```
Comment (t3chguy) [inline src/components/structures/auth/SoftLogout.js]: device/in

File: src/components/structures/auth/SoftLogout.js [hunk 5]
```diff
@@ -306,7 +306,7 @@ export default class SoftLogout extends React.Component {
                     <p>
                         {_t(
                             "Warning: Your personal data (including encryption keys) is still stored " +
-                            "on this device. Clear it if you're finished using this device, or want to sign " +
+                            "on this session. Clear it if you're finished using this session, or want to sign " +
```
Comment (t3chguy) [inline src/components/structures/auth/SoftLogout.js]: device

File: src/components/views/dialogs/ConfirmWipeDeviceDialog.js [hunk 6]
```diff
@@ -39,11 +39,11 @@ export default class ConfirmWipeDeviceDialog extends React.Component {
         return (
             <BaseDialog className='mx_ConfirmWipeDeviceDialog' hasCancel={true}
                         onFinished={this.props.onFinished}
-                        title={_t("Clear all data on this device?")}>
+                        title={_t("Clear all data on this session?")}>
```
Comment (t3chguy) [inline src/components/views/dialogs/ConfirmWipeDeviceDialog.js]: device

File: src/components/views/dialogs/ConfirmWipeDeviceDialog.js [hunk 7]
```diff
@@ -39,11 +39,11 @@ export default class ConfirmWipeDeviceDialog extends React.Component {
...
                 <div className='mx_ConfirmWipeDeviceDialog_content'>
                     <p>
                         {_t(
-                            "Clearing all data from this device is permanent. Encrypted messages will be lost " +
+                            "Clearing all data from this session is permanent. Encrypted messages will be lost " +
```
Comment (t3chguy) [inline src/components/views/dialogs/ConfirmWipeDeviceDialog.js]: device?

File: src/components/views/dialogs/SetPasswordDialog.js [hunk 8]
```diff
@@ -29,7 +29,7 @@ const WarmFuzzy = function(props) {
     if (props.didSetEmail) {
         title = _t('You have successfully set a password and an email address!');
     }
-    const advice = _t('You can now return to your account after signing out, and sign in on other devices.');
+    const advice = _t('You can now return to your account after signing out, and sign in on other sessions.');
```
Comment (t3chguy) [inline src/components/views/dialogs/SetPasswordDialog.js]: devices

File: src/components/views/dialogs/SetPasswordDialog.js [hunk 9]
```diff
@@ -114,7 +114,7 @@ export default createReactClass({
             >
                 <div className="mx_Dialog_content">
                     <p>
-                        { _t('This will allow you to return to your account after signing out, and sign in on other devices.') }
+                        { _t('This will allow you to return to your account after signing out, and sign in on other sessions.') }
```
Comment (t3chguy) [inline src/components/views/dialogs/SetPasswordDialog.js]: devices

File: src/components/views/right_panel/EncryptionInfo.js [hunk 10]
```diff
@@ -55,7 +55,7 @@ const EncryptionInfo = ({pending, member, onStartVerification}) => {
         <div className="mx_UserInfo_container">
             <h3>{_t("Verify User")}</h3>
             <div>
-                <p>{_t("For extra security, verify this user by checking a one-time code on both of your devices.")}</p>
+                <p>{_t("For extra security, verify this user by checking a one-time code on both of your sessions.")}</p>
```
Comment (t3chguy) [inline src/components/views/right_panel/EncryptionInfo.js]: devices

File: src/components/views/rooms/EventTile.js [hunk 11]
```diff
@@ -733,23 +733,23 @@ export default createReactClass({
             <div className="mx_EventTile_keyRequestInfo_tooltip_contents">
                 <p>
                     { this.state.previouslyRequestedKeys ?
-                        _t( 'Your key share request has been sent - please check your other devices ' +
+                        _t( 'Your key share request has been sent - please check your other sessions ' +
```
Comment (t3chguy) [inline src/components/views/rooms/EventTile.js]: devices?

File: src/components/structures/auth/SoftLogout.js (hunk 3 above)
Comment (jryans) [reply src/components/structures/auth/SoftLogout.js]: Changed to in

File: src/components/structures/auth/SoftLogout.js (hunk 4 above)
Comment (jryans) [reply src/components/structures/auth/SoftLogout.js]: Changed to in

File: src/components/structures/auth/SoftLogout.js (hunk 5 above)
Comment (jryans) [reply src/components/structures/auth/SoftLogout.js]: Changed to in

File: src/components/views/dialogs/ConfirmWipeDeviceDialog.js (hunk 6 above)
Comment (jryans) [reply src/components/views/dialogs/ConfirmWipeDeviceDialog.js]: Changed to in

File: src/components/views/dialogs/ConfirmWipeDeviceDialog.js (hunk 7 above)
Comment (jryans) [reply src/components/views/dialogs/ConfirmWipeDeviceDialog.js]: I think it's okay.

**Comment by jryans** [comment]
Yes, will do separately after locking and merging Weblate.

**Pull request merged**